### PR TITLE
Patient inheritance fix

### PR DESCRIPTION
**Pull request by holmesie**
Patients models no longer inherit from Data Element (fixes circular dependency issue). Also adjusted OID mapping for patient characteristics.

---

Pull requests into cqm-models require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass
- [ ] If there were any JavaScript changes, this PR has updated the `dist` directory using `npm run dist`
- [ ] If applicable, the library version number in `package.json` and `cqm-models.gemspec` has been updated

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: app/assets/javascripts/Patient.js [hunk 1]
```diff
@@ -24,9 +26,7 @@ const PatientSchema = DataElementSchema({
   // Cypress use a common field, it should be made a field on this model,
   // and not put into extendedData.
   extendedData: {},
-}, { id: false });
```
Comment (ljades) [inline app/assets/javascripts/Patient.js]: Why is this line getting deleted? We definitely need it for accessing the Patient Ids with a method.

File: data/oids.json [hunk 2]
```diff
@@ -374,7 +374,7 @@
   "patient_characteristic_clinical_trial_participant": {
     "hqmf_oid": "2.16.840.1.113883.10.20.28.3.6",
     "qrda_oid": "2.16.840.1.113883.10.20.24.3.51",
-    "category": "condition",
+    "category": "patient_characteristic",
```
Comment (okeefm) [inline data/oids.json]: Good catch
